CE09OSSM/CE09OSSM_R00018_ingest.csv

```diff
@@ -0,0 +1,53 @@
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CE09OSSM-MFD37-03-CTDBPE000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl37/CTDBP*/*ctdbp*.hex,CE09OSSM-MFD37-03-CTDBPE000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp.dcl.dosta_abcdjm_ctdbp_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl37/ctdbp2/*.ctdbp2.log,CE09OSSM-MFD37-03-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.dosta_abcdjm.ctdbp.dosta_abcdjm_ctdbp_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl37/CTDBP*/*ctdbp*.hex,CE09OSSM-MFD37-03-DOSTAD000,recovered_inst,Available,
+# ,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl37/camds/*.png,CE09OSSM-MFD37-06-CAMDSA000,recovered_host,Pending,No parser exists for this data set
+# ,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl37/CAMDS_*/*camds*/dcim/*.png,CE09OSSM-MFD37-06-CAMDSA000,recovered_inst,Pending,No parser exists for this data set
+mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl37/zplsc/*.zplsc.log,CE09OSSM-MFD37-07-ZPLSCC000,recovered_host,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl37/ZPLSC*/processed/*/*Averaged.nc,CE09OSSM-MFD37-07-ZPLSCC000,recovered_inst,Available,
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/cpm2/syslog/cpm_status.txt,CE09OSSM-RIC21-00-CPMENG000,recovered_host,Pending,Incorrectly specified dataset and parser
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl26/syslog/*.syslog.log,CE09OSSM-RID26-00-DCLENG000,recovered_host,Pending,Incorrectly specified dataset and parser
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl26/adcpt*/*.adcpt.log,CE09OSSM-RID26-01-ADCPTC000,recovered_host,Available,
+mi.dataset.driver.adcps_jln.adcps_jln_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl26/ADCPT*/*adcpt*.000,CE09OSSM-RID26-01-ADCPTC000,recovered_inst,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl26/velpt2/*.velpt2.log,CE09OSSM-RID26-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/instrmts/dcl26/VELPT*/*velpt*.aqd,CE09OSSM-RID26-04-VELPTA000,recovered_inst,Available,
+# mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CE09OSSM/R00018/cg_data/dcl26/phsen*/*.phsen*.log,CE09OSSM-RID26-06-PHSEND000,recovered_host,Not Deployed,Instrument not returned from vendor in time for deployment

```

Comment:
These are commented out saying "Not Deployed", but there is data (that looks valid) in these 2 PHSEN directories
omsds2: DS/CE09OSSM/R00018/cg_data/dcl26/phsen1/
omsds2: DS/CE09OSSM/R00018/instrmts/dcl26/PHSEN_snP0086/

Reply:
Corrected